src/playermat/Playermat.ttslua

```diff
@@ -1177,6 +1177,33 @@ function resetSlotSymbols()
   updateSlotSymbols()
 end
 
+function modifySlot(newSlot, cardPosition)
+  local slotSnaps = {}
+  local snaps = self.getSnapPoints()
+  for i, snap in ipairs(snaps) do
+    if inArea(snap.position, MAIN_PLAY_AREA) then
+      table.insert(slotSnaps, snap)
+    end
+  end
+  cardPosition.x = tonumber(string.format("%.3f", cardPosition.x))
+  cardPosition.z = tonumber(string.format("%.3f", cardPosition.z))
+  for snapId, snap in ipairs(slotSnaps) do
+    local snapPosX = tonumber(string.format("%.3f", snap.position.x))
+    local snapPosZ = tonumber(string.format("%.3f", snap.position.z))
+    if snapPosX == cardPosition.x and snapPosZ == cardPosition.z then
```

Comment:
```suggestion
  cardPosition.y = 0
  for snapId, snap in ipairs(slotSnaps) do
    local snapPosition = Vector(snap.position.x, 0, snap.position.z)
    if snapPosition == cardPosition then
```
I think this would be easier maintainable. This should work the same way since Vector have the additional property of being able to handle small rounding differences for this equality comparision (https://api.tabletopsimulator.com/vector/#arithmetics-summary)